packages/components/src/components/SegmentedControl/SegmentedControl.tsx

```diff
@@ -35,9 +36,11 @@ type SegmentProps = {
 
 const Segment = styled(Pressable)<SegmentProps>`
   border-radius: 8px;
-  padding-vertical: 7px;
+  padding-horizontal: ${spacing.vadsSpace2xs}px;
+  padding-vertical: ${spacing.vadsSpaceXs}px;
```

Comment:
Shouldn't the padding be 2xs (4) for both horizontal and vertical? 

![Screenshot 2024-09-20 at 10 07 16 AM](https://github.com/user-attachments/assets/577d16c4-2ec3-400e-9b07-e861f3655aca)

Reply:
Figma didn't get updated (cc: @jessicawoodin ), but we changed it in the [thread for the ticket](https://dsva.slack.com/archives/C07KFQ3U6QH/p1726684056537069?thread_ts=1726599619.164929&cid=C07KFQ3U6QH):
> But to answer your question, sure, let's go with xs/8 for the top/bottom padding. And I'll set the left/right padding to 2xs/4.

due to a misalignment of the existing component as implemented (5 vertical padding in Figma, 7 vertical padding in the implemented component) so adjusted to do what's the closest to current--and also kept things simpler by ensuring the touch target didn't become smaller than the 44 pixels tall.

Reply:
Also, could re-evaluate on #491 if desired. There were some other alignment issues with Segmented control between implemented component and Figma that could likely be squared away a bit. As our first component, it was a bit of a proving ground for other things and leaned on the existing app implementation perhaps more than it should've.